### PR TITLE
Update contact.class.php

### PR DESCRIPTION
**Pull request by dolibit-ut**
- 'Remove Roles' part was twice 
- translations and comments

### CODE REVIEW

**Comment by dolibit-ut** [comment]
I want to change birthday visible state from 3 to 1 

'birthday' =>array('type'=>'date', 'label'=>'Birthday', 'enabled'=>1, 'visible'=>3, 'position'=>80),

see  #23493 

Is there anything that speaks against this?

File: htdocs/contact/class/contact.class.php [hunk 1]
```diff
@@ -167,7 +167,10 @@ class Contact extends CommonObject
 	 */
 	public $state;
 
-	public $poste; // Position
+	/**
+	 * @var string // Job Position
```
Comment (eldy) [inline htdocs/contact/class/contact.class.php:171]: Avoid use of // when inside /*  */

File: htdocs/contact/class/contact.class.php [hunk 2]
```diff
@@ -176,7 +179,7 @@ class Contact extends CommonObject
 	public $fk_soc;		// both socid and fk_soc are used
 
 	/**
-	 * @var int 0=inactive, 1=active
+	 * @var int  0=inactive, 1=active // Status
```
Comment (eldy) [inline htdocs/contact/class/contact.class.php:182]: Avoid use of // when inside /*  */

File: htdocs/contact/class/contact.class.php [hunk 3]
```diff
@@ -1283,18 +1295,6 @@ public function delete($notrigger = 0)
 			}
 		}
 
-		if (!$error) {
-			// Remove Roles
-			$sql = "DELETE FROM ".MAIN_DB_PREFIX."societe_contacts WHERE fk_socpeople = ".((int) $this->id);
```
Comment (eldy) [inline htdocs/contact/class/contact.class.php:1288]: Why removing this ?
If we delete the contact, we must also delete it into this table, right ?

File: htdocs/contact/class/contact.class.php (hunk 1 above)
Comment (dolibit-ut) [reply htdocs/contact/class/contact.class.php:171]: okay

**Pull request closed without merging**